pkgs/development/python-modules/dm-tree/default.nix

```diff
@@ -48,8 +51,7 @@ buildPythonPackage rec {
   pythonImportsCheck = [ "tree" ];
 
   meta = with lib; {
-    broken = stdenv.isDarwin;
-    description = "Tree is a library for working with nested data structures";
+    description = "Tree is a library for working with nested data structures.";
```

Comment:
leave this as it was, esp no period. see https://ryantm.github.io/nixpkgs/stdenv/meta/#var-meta-description

Reply:
Done